src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMeanShort.cpp

```diff
@@ -0,0 +1,325 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Error.hpp>
+#include <OpenSpaceToolkit/Core/Types/Size.hpp>
+#include <OpenSpaceToolkit/Core/Utilities.hpp>
+
+#include <OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMeanShort.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+namespace orbit
+{
+namespace models
+{
+namespace kepler
+{
+
+using ostk::core::types::Size;
+using ostk::core::types::Integer;
+
+using EarthGravitationalModel = ostk::physics::environment::gravitational::Earth;
+
+BrouwerLyddaneMeanShort::BrouwerLyddaneMeanShort(
+    const Length &aSemiMajorAxis,
+    const Real &anEccentricity,
+    const Angle &anInclination,
+    const Angle &aRaan,
+    const Angle &anAop,
+    const Angle &aMeanAnomaly
+)
+    : BrouwerLyddaneMean(aSemiMajorAxis, anEccentricity, anInclination, aRaan, anAop, aMeanAnomaly)
+{
+}
+
+COE BrouwerLyddaneMeanShort::toCOE() const
+{
+    // ref:
+    // https://github.com/ChristopherRabotin/GMAT/blob/37201a6290e7f7b941bc98ee973a527a5857104b/src/base/util/StateConversionUtil.cpp#L3679
+
+    const auto mod = [](const Real &value, const Real &bound) -> Real
+    {
+        return value - bound * std::floor(value / bound);
+    };
+
+    // TBM: Make configurable?
+    const Real equatorialRadius = EarthGravitationalModel::EGM2008.equatorialRadius_.inMeters();
+
+    Integer pseudoState = 0;
+    bool possibleInaccuracyLongWritten = false;
+    bool brouwerSmallEccentricityWritten = false;
+
+    const Real re = EarthGravitationalModel::EGM2008.equatorialRadius_.inMeters();
+    const Real j2 = 1.082626925638815E-03;
+    const Real ae = 1.0;
+    Real smap = semiMajorAxis_.inMeters() / re;
+    Real eccp = eccentricity_;
+    Real incp = inclination_.inRadians();
+    Real raanp = raan_.inRadians(0.0, Real::TwoPi());
+    Real aopp = aop_.inRadians(0.0, Real::TwoPi());
+    Real meanAnomalyp = anomaly_.inRadians(0.0, Real::TwoPi());
+
+    if ((incp < 0.0) || (incp > Real::Pi()))
+    {
+        throw ostk::core::error::RuntimeError(String::Format(
+            "BrouwerLyddaneMeanShort is applicable only if 0.0 < inclination: [{}] < 180.0 degrees.",
+            Angle::Radians(incp).inDegrees()
+        ));
```

Comment:
```suggestion
        throw ostk::core::error::RuntimeError(
            "BrouwerLyddaneMeanShort is applicable only if 0.0 < inclination: [{}] < 180.0 degrees.",
            Angle::Radians(incp).inDegrees()
        );
```
nit: This should work. Runtime error has auto formatting